kivy/logger.py

```diff
@@ -115,8 +116,8 @@ def purge_logs(self, directory):
         if randint(0, 20) != 0:
             return
 
-        # Use config ?
-        maxfiles = 100
+        from kivy.config import Config
```

Comment:
OK, i will add the "￼if not maxfile > 0: return" line and replace all Config imports with a single one...